nautobot/dcim/views.py

```diff
@@ -1747,16 +1747,21 @@ class DeviceListView(generic.ObjectListView):
 
 class DeviceView(generic.ObjectView):
     queryset = Device.objects.select_related(
+        "cluster__cluster_group",
+        "controller_managed_device_group__controller",
+        "device_redundancy_group",
+        "device_type__device_family",
```

Comment:
Should we add `cluster`, `controller_managed_device_group`, and `device_type` in addition to the nested fields? Or is that already picked up by `select_related`, in that case, scratch this.